tests/integration_tests/config/simulate_prod_gamma_20_deg_South_check_output.yml

```diff
@@ -19,7 +19,7 @@ CTA_SIMPIPE:
       CORE_SCATTER: 10 700 m
       DATA_DIRECTORY: simtools-data
       OUTPUT_PATH: simtools-output
-      SIM_TELARRAY_SEEDS: 1745,290
+      SIM_TELARRAY_INSTRUMENT_SEEDS: 1745,290
```

Comment:
The only thing which I am not entirely happy about is that the second seed here is not only for the instrument (unlike the first one), so the name is a bit misleading. However, I think it's not a big deal and with a more detailed description of the argument it should be fine.

Reply:
OK. Should I keep it or change it?

Reply:
We can keep it